### PR TITLE
chore(deps): migrate to esa-oidc v4

### PR DESCRIPTION
**Pull request by derrabauke**
## Contents:
Migrate to ember-simple-auth-oidc v4. 
* 59fbc4b fix: protected route events
* 2e0521a chore(deps): upgrade ember-simple-auth-oidc

The contents of this PR start at 2e0521a
Commits before belong to #740

---

### CODE REVIEW

File: app/protected/template.hbs [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
     {{outlet}}
   </div>
 </main>
-
-<WelcomeModal
-  @visible={{this.visible}}
-  @on-never={{route-action 'neverTour'}}
-  @on-later={{route-action 'laterTour'}}
-  @on-start={{route-action 'startTour'}}
-}}
```
Comment (Yelinz) [inline app/protected/template.hbs:15]: Does the tour still work or will we not be showing it anymore?

Comment (derrabauke) [reply app/protected/template.hbs:15]: This block should stay in the code. Deleted by accident.

Comment (derrabauke) [reply app/protected/template.hbs]: Added again.

**Review by Yelinz — APPROVED** [review]
Rest looks good, i assume the tour still works

**Pull request merged**